src/test/java/lotto/domain/LottoTest.java

```diff
@@ -17,29 +17,37 @@ public class LottoTest {
     @Test
     @DisplayName("생성자 의존성 주입 - 로또 번호")
     void constructorDI() {
-        assertThat(new Lotto(lottoNumbers()).lottoNumbers().size()).isEqualTo(45);
+        assertThat(new Lotto(LottoNumber.lottoNumbers().subList(0,6)).lottoNumbers().size()).isEqualTo(6);
     }
 
     @Test
     @DisplayName("생성자 의존성 주입 - 당첨 번호 길이 예외")
-    void constructorDIExceptionForWinningNumberLength() {
+    void constructorDIExceptionForNumberLength() {
         assertThatThrownBy(() -> new Lotto(LottoNumber.lottoNumbers().subList(0, 7))).isInstanceOf(IllegalArgumentException.class);
     }
 
+    @Test
+    @DisplayName("생성자 의존성 주입 - 당첨 번호 중복 예외")
```

Comment:
```suggestion
    @DisplayName("중복되는 숫자로 로또 생성 시 예외가 발생한다")
```

이런 방식의 테스트 네이밍은 어떨까요? (당연히도 꼭 수정해 달라는 의미는 아니에요 😉)
또한 이 테스트와 바로 위의 테스트의 경우 테스트 이름에 "당첨 번호"가 들어가는데 실제로는 별로 상관이 없어 보이네요.